docs/api/apiv3/tags/baseline_comparisons.yml

```diff
@@ -0,0 +1,245 @@
+              },
+              {
+                "_meta": {
+                  "matchesFilters": true,
+                  "exists": true,
+                  "timestamp": "PT0S"
+                },
+                "_links": {
+                  "self": {
+                    "href": "/api/v3/work_packages/1528?timestamps=PT0S"
+                  }
+                }
+              }
+            ],
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "/api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S"
+      }
+    }
+  }
+  ```
+
+  ## Filter Matching
+
+  The work-packages API supports filtering the query results by one or several search criteria. See: [Filters](../filters)
+
+  To find out whether a work package matches the given set of filter criteria, check `"matchesFilters"` in the `"_meta"` section for each timestamp:
+
+  ```json
+  // /api/v3/work_packages?filters=...&timestamps=2022-01-01T00:00:00Z,PT0S
+  {
+    "_type": "WorkPackageCollection",
+    "_embedded": {
+      "elements": [
+        {
+          "_type": "WorkPackage",
+          "_embedded": {
+            "attributesByTimestamp": [
+              {
+                "_meta": {
+                  "matchesFilters": false,
+                  "timestamp": "2022-01-01T00:00:00Z"
+                }
+              },
+              {
+                "_meta": {
+                  "matchesFilters": true,
+                  "timestamp": "PT0S"
+                }
+              }
+            ],
+          }
+        }
+      ]
+    }
+  }
+  ```
+
+  In the above example, the work package matches the filter criteria at the timestamp `"PT0S"`, but does not match the filter criteria at the timestamp `"2022-01-01T00:00:00Z"`. The work package is included in the returned collection because it matches the filter criteria at least at one of the given timestamps.
+
+  ## Existence
+
+  To find out whether a work package has existed at the given timestamps, check `"exists"` in the `"_meta"` section for each timestamp:
+
+  ```json
+  // /api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S
+  {
+    "_type": "WorkPackageCollection",
+    "_embedded": {
+      "elements": [
+        {
+          "_type": "WorkPackage",
+          "_embedded": {
+            "attributesByTimestamp": [
+              {
+                "_meta": {
+                  "exists": false,
+                  "timestamp": "2022-01-01T00:00:00Z"
+                }
+              },
+              {
+                "_meta": {
+                  "exists": true,
+                  "timestamp": "PT0S"
+                }
+              }
+            ],
+          }
+        }
+      ]
+    }
+  }
+  ```
+
+  In the above example, the work package exists at the timestamp `"PT0S"`, but has not existed at the timestamp `"2022-01-01T00:00:00Z"`. The work package is included in the returned collection because it has existed at least at one of the given timestamps.
```

Comment:
I again don't think it waisted to explicitly point out that it could be the other way around. Although, that would require a weird kind of timestamps as we don't support soft deletion. So maybe only the lacking support of the soft deletion should be noted here.

Reply:
➝ https://github.com/opf/openproject/pull/11783/commits/5a33ab64d384e89c6da23412661af7e4aea2182e